src/pages/understand-stacks/regtest.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Regtest
+description: Test your smart contracts and apps
+images:
+  large: /images/pages/regtest.svg
+  sm: /images/pages/regtest-sm.svg
+---
+
+## About regtest
```

Comment:
> However, it differs by producing a new BTC and STX block every two minutes, making it much more suitable for rapid development. The regtest is reset frequently.

could we add to the testnet page how often it produces blocks and how often it will be reset? That's not clear, do devs woulnd't know which one to use